<ui version="4.0" >
 <class>UIPluginsSettings</class>
 <widget class="QDialog" name="UIPluginsSettings" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Plugins Settings</string>
  </property>
  <layout class="QGridLayout" >
   <property name="leftMargin" >
    <number>5</number>
   </property>
   <property name="topMargin" >
    <number>5</number>
   </property>
   <property name="rightMargin" >
    <number>5</number>
   </property>
   <property name="bottomMargin" >
    <number>5</number>
   </property>
   <property name="horizontalSpacing" >
    <number>3</number>
   </property>
   <property name="verticalSpacing" >
    <number>3</number>
   </property>
   <item row="2" column="0" colspan="2" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>551</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="2" >
    <widget class="QPushButton" name="pbClose" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text" >
      <string>Close</string>
     </property>
    </widget>
   </item>
   <item rowspan="2" row="0" column="0" >
    <layout class="QVBoxLayout" >
     <property name="spacing" >
      <number>2</number>
     </property>
     <property name="leftMargin" >
      <number>0</number>
     </property>
     <property name="topMargin" >
      <number>0</number>
     </property>
     <property name="rightMargin" >
      <number>0</number>
     </property>
     <property name="bottomMargin" >
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="lType1" >
       <property name="text" >
        <string>Plugin Type :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbType" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Minimum" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize" >
        <size>
         <width>120</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lName1" >
       <property name="text" >
        <string>Plugin Name :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="lwNames" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Expanding" hsizetype="Maximum" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="1" colspan="2" >
    <widget class="QStackedWidget" name="swWidgets" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
       <horstretch>0</horstretch>
       <verstretch>99</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape" >
      <enum>QFrame::Box</enum>
     </property>
     <widget class="QWidget" name="page" />
     <widget class="QWidget" name="page_2" />
    </widget>
   </item>
   <item row="0" column="1" colspan="2" >
    <widget class="QGroupBox" name="gbInformations" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Maximum" hsizetype="Preferred" >
       <horstretch>0</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="title" >
      <string>Informations</string>
     </property>
     <layout class="QGridLayout" >
      <property name="leftMargin" >
       <number>3</number>
      </property>
      <property name="topMargin" >
       <number>3</number>
      </property>
      <property name="rightMargin" >
       <number>3</number>
      </property>
      <property name="bottomMargin" >
       <number>3</number>
      </property>
      <property name="horizontalSpacing" >
       <number>5</number>
      </property>
      <property name="verticalSpacing" >
       <number>5</number>
      </property>
      <item row="1" column="1" >
       <widget class="QCheckBox" name="cbEnabled" >
        <property name="text" >
         <string>Enabled</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QCheckBox" name="cbEnableUponStart" >
        <property name="text" >
         <string>Enable upon start</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" >
       <layout class="QGridLayout" >
        <property name="leftMargin" >
         <number>0</number>
        </property>
        <property name="topMargin" >
         <number>0</number>
        </property>
        <property name="rightMargin" >
         <number>0</number>
        </property>
        <property name="bottomMargin" >
         <number>0</number>
        </property>
        <property name="horizontalSpacing" >
         <number>3</number>
        </property>
        <property name="verticalSpacing" >
         <number>3</number>
        </property>
        <item row="1" column="0" >
         <widget class="QLabel" name="lName" >
          <property name="text" >
           <string>Name :</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1" >
         <widget class="QLineEdit" name="leType" >
          <property name="readOnly" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="3" column="0" >
         <widget class="QLabel" name="lType" >
          <property name="text" >
           <string>Type :</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1" >
         <widget class="QLineEdit" name="leName" >
          <property name="readOnly" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="1" >
         <widget class="QLineEdit" name="leVersion" >
          <property name="readOnly" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="1" >
         <widget class="QLineEdit" name="leCaption" >
          <property name="readOnly" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="0" >
         <widget class="QLabel" name="lCaption" >
          <property name="text" >
           <string>Caption :</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0" >
         <widget class="QLabel" name="lVersion" >
          <property name="text" >
           <string>Version :</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="1" >
       <layout class="QVBoxLayout" >
        <property name="spacing" >
         <number>3</number>
        </property>
        <property name="leftMargin" >
         <number>0</number>
        </property>
        <property name="topMargin" >
         <number>0</number>
        </property>
        <property name="rightMargin" >
         <number>0</number>
        </property>
        <property name="bottomMargin" >
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="lDescription" >
          <property name="text" >
           <string>Description :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="teDescription" >
          <property name="readOnly" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pbClose</sender>
   <signal>clicked()</signal>
   <receiver>UIPluginsSettings</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>602</x>
     <y>299</y>
    </hint>
    <hint type="destinationlabel" >
     <x>330</x>
     <y>158</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
